flutter/lib/resources/configurations_manager.dart

```diff
@@ -21,12 +22,46 @@ class BenchmarksConfig {
 
 class ConfigurationsManager {
   final String applicationDirectory;
+  final ResourceManager resourceManager;
   final BenchmarksConfig defaultConfig =
       BenchmarksConfig('default', _defaultConfigUrl);
+  String currentConfigName;
+  String configPath = '';
 
-  ConfigurationsManager(this.applicationDirectory);
+  ConfigurationsManager(
```

Comment:
Maybe rename this class to `ConfigManager` to be consistent with using the short form.

Reply:
done